ophyd/areadetector/filestore_mixins.py

```diff
@@ -169,6 +168,43 @@ def stage(self):
         self._resource = fs.insert_resource('AD_TIFF', self._fp, res_kwargs)
 
 
+class FileStoreTIFFSquashing(FileStoreBase):
```

Comment:
I know `examples/` is filled with junk currently, but examples of these concepts would be a good fit there for when it finally gets cleaned up.